alpine/Dockerfile

```diff
@@ -33,7 +33,9 @@ RUN set -ex; \
 	chown kong:0 /usr/local/bin/kong && \
```

Comment:
Answering to your question down there, this line will fail if the file doesn't exist, so it's already a `-f /usr/local/bin/kong`